build/charts/antrea/crds/clusternetworkpolicy.yaml

```diff
@@ -604,6 +604,17 @@ spec:
                               type: string
                             namespace:
                               type: string
+                      toMultiClusterServices:
+                        type: array
+                        items:
+                          type: object
+                          required:
+                            - name
+                          properties:
```

Comment:
why namespace is not required?

Reply:
Reason is when this field is used with ANP, user can choose to not provide a namespace for the service and the namespace of the ANP will be assumed

Reply:
My bad, this is for ACNP only. Fixed